src/libraries/System.Net.Requests/src/System/Net/TimerThread.cs

```diff
@@ -471,7 +471,7 @@ private static void Prod()
 
             if (oldState == TimerThreadState.Idle)
             {
-                new Thread(new ThreadStart(ThreadProc)).Start();
+                new Thread(new ThreadStart(ThreadProc)) { IsBackground = true, Name = ".NET Net Timer" }.Start();
```

Comment:
`.NET Net` -> `.NET` ?

Reply:
It's used by networking - is not the regular timer thread. Trying to contract length. I can change back

Reply:
I have not realized that "Net" is meant to be shortcut for networking here. I do not have a strong opinion on this one. System.Net.Request is legacy library that we do not expect to be used for anything that matters.